ci-bench-runner/migrations/0_create.sql

```diff
@@ -17,7 +18,8 @@ CREATE TABLE comparison_runs(
     created_utc TEXT NOT NULL,
     baseline_commit TEXT NOT NULL,
     candidate_commit TEXT NOT NULL,
-    scenarios_missing_in_baseline TEXT
+    icount_scenarios_missing_in_baseline TEXT,
+    walltime_scenarios_missing_in_baseline TEXT

```

Comment:
The changes in this file feel like a case where we should be using the migration support to apply a delta over the pre-existing `0_create.sql`. I think with this approach we'd have to drop the production database and recreate it from scratch right?

Reply:
We don't have a production database yet, so I think we can get away with this. Or is there a different reason why this might be a problem after all?

By the way, SQLx will raise a fatal error if a migration was modified after it was applied (nice to know SQLx has our back and will prevent accidents even if we mess things up).

Reply:
> We don't have a production database yet, so I think we can get away with this. Or is there a different reason why this might be a problem after all?

Nope, I think I just tricked myself into thinking there was an instance on the dedicated server already and so it made sense to treat this as a migration layered on top :sweat_smile: 

>By the way, SQLx will raise a fatal error if a migration was modified after it was applied (nice to know SQLx has our back and will prevent accidents even if we mess things up).

Oh great :+1: That does help alleviate my concern here.